src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -412,12 +414,13 @@ namespace
             }
         }
 
-        static double compute_sun_radius(const double distance)
+        static double compute_sun_radius(const double distance, const double m_safe_scene_radius)
```

Comment:
I wonder if there is a better way.

Reply:
The `m_` prefix means *member variable*, it should only be used for member variables, not for arguments.

Reply:
Another very confusing thing: when you're calling `compute_sun_radius()`, you're passing `m_safe_scene_diameter` in 2nd argument, but the argument itself is called `m_safe_scene_radius`.